pipelines/_ingredients/LSSTComCam/DRP.yaml

```diff
@@ -75,7 +75,8 @@ tasks:
     class: lsst.analysis.tools.tasks.refCatSourceAnalysis.RefCatSourceAnalysisTask
     config:
       # Only run metrics for analyzing the preSources:
-      connections.data: preSourceTable_visit_gaia_dr3_20230707_match_astrom
+      connections.data: preSourceTable_visit_uw_stars_20240524_match_astrom
```

Comment:
Same as LSSTCam comment, what was here before should be correct.

Reply:
Changed back to `preSourceTable_visit_gaia_dr3_20230707_match_astrom`.